index.js

```diff
@@ -2369,6 +2383,20 @@
 
   //. ### Object
 
+  //# prop :: Accessible a => String -> a -> b
+  //.
+  //. Takes a property name and an object and returns the value of the
+  //. specified field. If for some reason the record lacks the specified field,
+  //. a type error is thrown.
```

Comment:
I suggest two tweaks:

  - "object with known properties" in place of "object"; and
  - "property" in place of "field" (since we no longer mention records).